environment.yml

```diff
@@ -1,319 +1,9 @@
-  - pywavelets=1.1.1=py37h1de35cc_0
-  - pyyaml=5.3.1=py37haf1e3a3_1
-  - pyzmq=19.0.1=py37hb1e8313_1
-  - qdarkstyle=2.8.1=py_0
-  - qt=5.9.7=h468cd18_1
-  - qtawesome=0.7.2=py_0
-  - qtconsole=4.7.5=py_0
-  - qtpy=1.9.0=py_0
-  - readline=8.0=h1de35cc_0
-  - regex=2020.6.8=py37haf1e3a3_0
-  - requests=2.24.0=py_0
-  - ripgrep=11.0.2=he32d670_0
-  - rope=0.17.0=py_0
-  - rtree=0.9.4=py37_1
-  - ruamel_yaml=0.15.87=py37haf1e3a3_1
-  - scikit-image=0.16.2=py37h6c726b0_0
-  - scikit-learn=0.23.1=py37h603561c_0
-  - scipy=1.5.0=py37h912ce22_0
-  - seaborn=0.10.1=py_0
-  - send2trash=1.5.0=py37_0
-  - setuptools=49.2.0=py37_0
-  - simplegeneric=0.8.1=py37_2
-  - singledispatch=3.4.0.3=py37_0
-  - sip=4.19.8=py37h0a44026_0
-  - six=1.15.0=py_0
-  - snappy=1.1.8=hb1e8313_0
-  - snowballstemmer=2.0.0=py_0
-  - sortedcollections=1.2.1=py_0
-  - sortedcontainers=2.2.2=py_0
-  - soupsieve=2.0.1=py_0
-  - sphinx=3.1.2=py_0
-  - sphinxcontrib=1.0=py37_1
-  - sphinxcontrib-applehelp=1.0.2=py_0
-  - sphinxcontrib-devhelp=1.0.2=py_0
-  - sphinxcontrib-htmlhelp=1.0.3=py_0
-  - sphinxcontrib-jsmath=1.0.1=py_0
-  - sphinxcontrib-qthelp=1.0.3=py_0
-  - sphinxcontrib-serializinghtml=1.1.4=py_0
-  - sphinxcontrib-websupport=1.2.3=py_0
-  - spyder=4.1.4=py37_0
-  - spyder-kernels=1.9.2=py37_0
-  - sqlalchemy=1.3.18=py37haf1e3a3_0
-  - sqlite=3.32.3=hffcf06c_0
-  - statsmodels=0.11.1=py37haf1e3a3_0
-  - sympy=1.6.1=py37_0
-  - tbb=2020.0=h04f5b5a_0
-  - tblib=1.6.0=py_0
-  - terminado=0.8.3=py37_0
-  - testpath=0.4.4=py_0
-  - threadpoolctl=2.1.0=pyh5ca1d4c_0
-  - tk=8.6.10=hb0a8c7a_0
-  - toml=0.10.1=py_0
-  - toolz=0.10.0=py_0
+  - python=3.7.7
+  - pytorch=1.6.0
   - torchvision=0.7.0=py37_cpu
-  - tornado=6.0.4=py37h1de35cc_1
-  - tqdm=4.47.0=py_0
-  - traitlets=4.3.3=py37_0
-  - typed-ast=1.4.1=py37h1de35cc_0
-  - typing_extensions=3.7.4.2=py_0
-  - ujson=1.35=py37h1de35cc_0
-  - unicodecsv=0.14.1=py37_0
-  - unixodbc=2.3.7=h1de35cc_0
-  - urllib3=1.25.9=py_0
-  - watchdog=0.10.3=py37haf1e3a3_0
-  - wcwidth=0.2.5=py_0
-  - webencodings=0.5.1=py37_1
-  - werkzeug=1.0.1=py_0
-  - wheel=0.34.2=py37_0
-  - widgetsnbextension=3.5.1=py37_0
-  - wrapt=1.11.2=py37h1de35cc_0
-  - wurlitzer=2.0.1=py37_0
-  - xlrd=1.2.0=py37_0
-  - xlsxwriter=1.2.9=py_0
-  - xlwings=0.19.5=py37_0
-  - xlwt=1.3.0=py37_0
-  - xz=5.2.5=h1de35cc_0
-  - yaml=0.2.5=haf1e3a3_0
-  - yapf=0.30.0=py_0
-  - zeromq=4.3.2=hb1e8313_2
-  - zict=2.0.0=py_0
-  - zipp=3.1.0=py_0
-  - zlib=1.2.11=h1de35cc_3
-  - zope=1.0=py37_1
-  - zope.event=4.4=py37_0
-  - zope.interface=4.7.1=py37h1de35cc_0
-  - zstd=1.4.5=h41d2c2f_0
-  - pip:
-    - blis==0.4.1
-    - catalogue==1.0.0
-    - cymem==2.0.3
-    - de-core-news-sm==2.3.0
-    - en-core-web-sm==2.3.1
-    - murmurhash==1.0.2
-    - plac==1.1.3
-    - preshed==3.0.2
-    - sentencepiece==0.1.91
-    - spacy==2.3.2
-    - srsly==1.0.2
-    - thinc==7.4.1
-    - torchtext==0.7.0
-    - wasabi==0.8.0
-prefix: /Users/DivyanshuMurli1/anaconda3/envs/pytorch_updated
```

Comment:
Best to keep this line out - it contains specific info about your computer filesystem, which is kindof private